<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cainapp2</class>
 <widget class="QWidget" name="cainapp2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cainapp2</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:24pt; font-weight:600;&quot;&gt;CAIN APP 2.0&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>130</y>
     <width>231</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>211</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Input fps</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-140</x>
     <y>50</y>
     <width>1031</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>80</y>
     <width>231</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;other&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>211</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Interpolation factor</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>gpu id</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>2x</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4x</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboBox_6">
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="checkBox_7">
       <property name="text">
        <string>multi gpu mode</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="checkBox_4">
       <property name="text">
        <string>half precision</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spin_3">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>batch_size</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>481</width>
     <height>341</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="midLineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;input and output&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>461</width>
      <height>200</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="2">
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>mp4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mov</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mkv</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="checkBox_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>use prores</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>model</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QComboBox" name="comboBox_3">
       <item>
        <property name="text">
         <string>lt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>standard</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hq</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4444</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4444xq</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QComboBox" name="comboBox_4"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="comboBox_5">
       <item>
        <property name="text">
         <string>png</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>bmp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>tga</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>jp2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>jpg</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>output ffmpeg line:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>input ffmpeg line:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>output video type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="text">
        <string>-c:v h264_nvenc -b:a 300k -cq 10</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>temp frame sequence file type</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>459</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select output path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>459</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Select input video</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>170</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CheckBox</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>210</y>
     <width>102</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>vr mode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Extract frames</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Interpolate!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>430</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>All steps</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>430</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>encode video from frames</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
